wordle-web/pages/leaderboard.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+    <!--create a leader board that shows top scores-->
+    <v-card>
+        <v-card-title class = "text-center">Leader Board</v-card-title>
+        <v-table>
+            <thead>
+                <tr>
+                    <th>User Name</th>
+                    <th>Average Attempts</th>
+                    <th>Game Count</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr v-for="score in scoresToDisplay" :key="score.name">
+                    <td v-if="score.name">{{ score.name }}</td>
+                    <td v-if="score.averageAttempts">{{ score.averageAttempts }}</td>
+                    <td v-if="score.gameCount">{{ score.gameCount }}</td>
+                    
+                </tr>
+            </tbody>
+        </v-table>
+        <v-card-actions>
+            <v-btn @click="router.push('/')">Back</v-btn>
```

Comment:
```suggestion
      <v-btn color="primary" variant="flat" @click="router.push('/')">Back</v-btn>
```
This button and the one one on `about` page are styled differently but in the same place and have the same action so you should style them the same

Reply:
Nice catch, we definitely should do that.